src/test/java/edu/hm/hafner/analysis/parser/CodeCheckerParserWithWindowsPathsTest.java

```diff
@@ -0,0 +1,73 @@
+package edu.hm.hafner.analysis.parser;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.assertions.SoftAssertions;
+
+import static edu.hm.hafner.analysis.assertions.Assertions.*;
+
+class CodeCheckerParserWithWindowsPathsTest extends AbstractParserTest {
```

Comment:
Please combine this test case with the other one in a single file. You can add an additional test method. 
Example: https://github.com/jenkinsci/analysis-model/blob/7805d15423612fe68494f63a89d4bce6a96ce514/src/test/java/edu/hm/hafner/analysis/parser/PylintParserTest.java#L255